posthog/settings/dynamic_settings.py

```diff
@@ -108,6 +108,7 @@
         "Used to disable emails from async migrations service",
         bool,
     ),
+    "INGESTION_SITE_URL": (None, "Used in ingestion pipeline to determine sites url", str),
```

Comment:
do we really need this? I can see how messing with `SITE_URL` might be annoying but I feel this is confusing to users - they will be very confused as to why there's `SITE_URL` and `INGESTION_SITE_URL` and they're both set differently.

Reply:
Note this is not shown in instance settings. :)

Reply:
True - got confused quickly reading here. Point still stands but less relevant

Reply:
Not sure what that point is. INGESTION_SITE_URL is purely managed by plugin-server and is a setting we're using to do this whole dance.